### DISCUSSION TITLE
Improve collection navigation settings

### DISCUSSION THREAD

**Discussion by pierre-H** [directus/directus#5871] [Feature Requests]
In the navigation collection setting of a role, we can improve the select field : 
1. by hiding collections for which the option *Hide within the App* is true
2. by adding in gray the name on the sql table. Example : **Clients** (project_clients) where _Clients_ is the translation of the collection and _project_clients_ is the sql table name.

Also, allowing to set translations for the group name would be great.

**Comment by azrikahar** [comment]
> 2\. by adding in gray the name on the sql table. Example : **Clients** (project_clients) where _Clients_ is the translation of the collection and _project_clients_ is the sql table name.

Just to add an additional info before closing this for now, #13103 (released in `9.11.0`) has since included a tooltip to show the sql table name!

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
